sources/packages/web/src/views/aest/institution/Programs.vue

```diff
@@ -128,16 +98,17 @@ export default defineComponent({
       {} as PaginatedResults<EducationProgramsSummary>,
     );
     const searchProgramName = ref("");
-    const currentPageSize = ref();
-    const loading = ref(false);
+    const currentPage = ref();
+    const currentPageLimit = ref();
+    const loading = ref(true);
 
     const getProgramsSummaryList = async (
       institutionId: number,
-      rowsPerPage: number,
-      page: number,
+      rowsPerPage = DEFAULT_PAGE_LIMIT,
+      page = DEFAULT_DATATABLE_PAGE_NUMBER,
```

Comment:
Please revert this twi line to avoid the Sonarcloud issue.

Reply:
I made those 2 as the last parameters. Does that work?